code/modules/station_map/station_map.dm

```diff
@@ -23,14 +23,19 @@
 		icon = map_icon
 
 	Click(location, control, params)
-		if (!isAIeye(usr)) //no humans teleporting
+		if (!isAI(usr)) //only for AI use
 			return
 		var/list/param_list = params2list(params)
 		if ("left" in param_list)
 			var/x = text2num(param_list["icon-x"])
 			var/y = text2num(param_list["icon-y"])
 			var/turf/clicked = locate(x, y, Z_LEVEL_STATION)
-			usr.loc = clicked
+			if (isAIeye(usr))
```

Comment:
Handling the player in both mainframe and AI Eye states here; the mainframe code path here is reliant on the isAI conditional only having two states. I would prefer to use an explicit check like isMainframe, but that is not what I expected it to be and I don't want to bloat scope.